test/verify/check-machines

```diff
@@ -370,6 +380,71 @@ class TestMachines(MachineCase):
             print("Libvirt version does not support disk statistics")
             b.wait_present("#vm-{0}-disksstats-unsupported".format(name))
 
+    def testLibvirt(self):
+        b = self.browser
+        m = self.machine
+
+        libvirtServiceName = self.getLibvirtName()
+
+        def checkLibvirtEnabled():
+            try:
+                m.execute("systemctl -q is-enabled {0}".format(libvirtServiceName))
+                return True
+            except:  # exception if return code != 0
+                return False
+
+        self.startVm("subVmTest1")
+        self.login_and_go("/machines")
+
+        b.wait_in_text("body", "Virtual Machines")
+        b.wait_present("tbody tr th")
+        b.wait_in_text("tbody tr th", "subVmTest1")
+
+        b.wait(lambda: checkLibvirtEnabled())
+        m.execute("systemctl disable {0}".format(libvirtServiceName))
+        print(self.getLibvirtName())
```

Comment:
Debugging leftover. Also this shouldn't be necessary as when running the tests with `-vt` it already shows you all the commands and their outputs.